doc/wallet-passphrase.md

```diff
@@ -0,0 +1,21 @@
+## Wallet Passphrase
+
+Understanding the nuances of wallet security is essential to storing your Bitcoin safely. One crucial aspect of safeguarding funds is the utilization of a wallet passphrase or password, implemented through wallet encryption. Let's explore key points to demystify the nature of a wallet passphrase and the encryption process, emphasizing what they do and what they don't do:
+
+  - **Not the Seed:**
+The wallet passphrase is distinct from the seed. The seed, also known as the HD seed, functions as a master key for deriving key pairs in a hierarchical deterministic (HD) wallet. The passphrase, on the other hand, serves as an additional layer of security, specifically designed to secure the private keys within the wallet. While the HD seed is essential for wallet restoration and fund recovery, the passphrase serves as a safeguard, demanding an additional layer of authentication even if the seed is compromised. This dual-layered approach enhances overall wallet security, effectively mitigating risks associated with unauthorized access and potential theft.
```

Comment:
We should avoid portraying the seed as something that is recommended for restoring a wallet. The seed is not easily exported and doing so is not a recommended method of backing up a wallet. Also not all encrypted wallets will have a seed, and not all private keys in a wallet are necessarily derived from the same seed, if derived at all.

Reply:
Fixed